src/frontend/src/components/base/RadioButtonGroup/RadioButtonGroup.styles.ts

```diff
@@ -0,0 +1,20 @@
+import { tv } from "tailwind-variants";
+
+export const radioButtonGroupStyles = tv({
+    base: "flex gap-3",
+});
+
+export const radioButtonStyles = tv({
+    base: [
+        "flex items-center justify-center rounded-[var(--spacing-4)] border px-4 py-2 h-[34px] gap-3 self-stretch flex-1",
+        "text-gray-800 text-sm font-bold transition-colors border",
+        "cursor-pointer focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-muted-blue-400 focus-visible:ring-offset-2",
+        "pressed:scale-[0.98]",
+    ],
+    variants: {
+        isSelected: {
+            true: "border-gray-500",
+            false: "border-gray-300 hover:border-gray-500",
+        },
+    },
```

Comment:
I did no add Small, Medium, Large sizes in the style definitions, since there seems to be only one style in the UI design, which is only used in the mobile view. We could customize size further as we encounter them later, I'd suggest.